Use more inclusive pronouns (#1852)
nextcloud/documentation: admin_manual/configuration_server/occ_command.rst

 sudo -u www-data php occ dav:create-calendar molly mollycal
 

> Molly will immediately see these on their Calendar and Contacts pages.

Reviewer: For all the sharing-related topics, I see a lot of potential for confusion: Molly wants to see her calendar on her Calendar and Contacts page and she doesn't want to see her calendar on the Calendar or Contacts page of anyone else (unless she shared it, or is it shared by default?).

Reviewer 2: Here’s a better sentence to remove confusion:
```suggestion
Molly will immediately see these in the Calendar and Contacts apps.
```
